tests/python/utils.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+from torch.testing._internal.common_utils import TEST_WITH_ROCM
+from torch.testing._internal.jit_utils import RUN_CUDA
+
+# flake8: noqa
+from nvfuser import FusionDefinition, DataType
+
+RUN_NVFUSER = RUN_CUDA and not TEST_WITH_ROCM
```

Comment:
do we still need these? I'm pretty sure these are coming from the torchscript tests in the old days.